### PR TITLE
Match uploader tweaks

### PR DESCRIPTION
**Pull request by mojodna**
The updated upload status endpoint includes GeoJSON, which is easier to deal with.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-8.2.1
+8
```
Comment (tombh) [inline .nvmrc]: How come?

**Comment by mojodna** [comment]
This complements changes in https://github.com/hotosm/oam-catalog/pull/104

File: .nvmrc (hunk 1 above)
Comment (mojodna) [reply .nvmrc]: To pick up compatible changes w/o needing to track Node releases closely and changing this when they occur. Pin it to `8.2` if you want, but locking to a patch release prevents bug / security fixes from being picked up.

Comment (mojodna) [reply .nvmrc]: Alternately, is there a reason that specifically 8.2.1 needs to be used?

Comment (tombh) [reply .nvmrc]: Isn't this the same conversation that led to `npm` finally adding `package-lock.json`? Sure the flexibility of `^1`-style versioning makes security patching "easier" (you still have to rebuild the project, it's not total magic), but you lose out on the main benefit of version pinning: behaviour guarantees. Which is why `npm` relented and made `package-lock.json` default in v5.

So `8.2.1` is a guarantee of behaviour and passing tests.

**Comment by tombh** [comment]
So you'll need to;

1) Bump the version in oam-catalog's package.json
2) Bump the reference to the oam-catalog version in this repo's package.json
3) Git tag oam-catalog to the new version

**Comment by mojodna** [comment]
I'll leave those to you after merging.

File: .nvmrc (hunk 1 above)
Comment (mojodna) [reply .nvmrc]: Feel free to revert this.

**Comment by mojodna** [comment]
No longer necessary now that the updated `oam-catalog` is writing WKT footprints.

**Pull request closed without merging**